backend/src/main/java/com/hyooun/wilson/project/domain/Project.java

```diff
@@ -0,0 +1,20 @@
+package com.hyooun.wilson.project.domain;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+
+@Getter
+@RequiredArgsConstructor
+public class Project {
+
+	private final ProjectId id;
+	private final String name;
+
+	// 향후 필요 시 GitHub repo, 연결된 사용자, 생성일자 등도 추가 가능
+
+	@Value
```

Comment:
Project 클래스에 ProjectId VO 추가 필요.

```suggestion
public static class ProjectId { Long value; }
```